gxa/src/main/javascript/atlas_bundles/bioentity-information/index.js

```diff
@@ -1 +1 @@
-module.exports = require('./src/renderer.js');
+module.exports = require('sc-atlas-bioentity-information');
```

Comment:
Use ES modules. Any CommonJS module stuff around is now legacy code. [Read more about modules here](https://auth0.com/blog/javascript-module-systems-showdown/).